src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -80,5 +81,28 @@ public async Task<ActionResult> CreateUserAsync(Guid organisationId, CreateUserR
 
             return Ok();
         }
+
+        [Route("users/{userId}")]
+        [HttpGet]
+        public async Task<ActionResult> GetUserById(string userId)
+        {
+            var user = await _usersRepository.GetUserById(userId);
+
+            if (user is null)
+            {
+                return NotFound();
+            }
+
+            var userViewModel = new GetUserViewModel()
```

Comment:
The brackets can be omitted here (they're not required with object initializers).